app/models/item.rb

```diff
@@ -43,4 +43,12 @@ def toggle
     end
     self
   end
+
+  def self.most_popular_items
+    joins(:item_orders).where(active?: true).group(:id).select("items.*, sum(quantity) as quantity_purchased").order("quantity_purchased DESC").limit(5)
```

Comment:
Is this able to join the Items table to the item_orders table because it is an instance method?